src/components/Icon/Icon.tsx

```diff
@@ -5,68 +5,149 @@ import svg4everybody from 'svg4everybody';
 import styles from './Icon.module.css';
 import icons from '../../icons/spritemap/spritemap.svg';
 
-export interface Props {
+interface IconPropsBase {
   /**
    * CSS class names that can be appended to the component.
    */
   className?: string;
   /**
-   * Declares whether the icon is focusable or not for accessibility
+   * Either a fragment of svg elements (g, path, circle, etc.), or a single
+   * svg element. Useful for creating specific icon components.
+   *
+   * @example
+   * function CircleIcon(props: IconProps) {
+   *   return (
+   *     <Icon {...props}>
+   *       <circle cx="50" cy="50" r="50" />
+   *     </Icon>
+   *   )
+   * }
    */
-  focusable?: boolean;
+  children?: React.ReactNode;
+  /**
+   * The SVG Color, expects a valid css color (hex, rgb, etc.).
+   * Recommendation: if `currentColor` isn't sufficient, use color tokens from
+   * @chanzuckerberg/eds/lib/tokens/colors.ts
```

Comment:
hmmmm I guess these should surface the tier-2 color tokens instead. will add that as a ticket